src/test/java/com/codurance/retropolis/factories/CardFactoryTest.java

```diff
@@ -11,18 +11,11 @@
   @Test
   void returns_a_new_card() {
     NewCardRequestObject newCardRequestObject = new NewCardRequestObject("hello", 1, "John Doe");
-    CardFactory cardFactory = new TestableCardFactory();
+    CardFactory cardFactory = new CardFactory();
```

Comment:
Cool. So we no longer need the TestableCardFactory because we are not using the static method to get the Board. The id for the Card will be created automatically in the SQL.